extensions/mssql/src/objectManagement/ui/serverPropertiesDialog.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import { ObjectManagementDialogBase, ObjectManagementDialogOptions } from './objectManagementDialogBase';
+import { IObjectManagementService } from 'mssql';
+import * as localizedConstants from '../localizedConstants';
+import { ViewServerPropertiesDocUrl } from '../constants';
+import { ServerPropertiesInfo, ServerPropertiesViewInfo } from '../interfaces';
+
+export class ServerPropertiesDialog extends ObjectManagementDialogBase<ServerPropertiesInfo, ServerPropertiesViewInfo> {
+	private generalTab: azdata.window.DialogTab;
+	private generalSection: azdata.GroupContainer;
+	private nameInput: azdata.InputBoxComponent;
+	private languageDropdown: azdata.DropDownComponent;
+	private memoryInput: azdata.InputBoxComponent;
+	private operatingSystemInput: azdata.InputBoxComponent;
+
+	private memoryTab: azdata.window.DialogTab;
+	private memorySection: azdata.GroupContainer;
+	private minServerMemoryInput: azdata.InputBoxComponent;
+	private maxServerMemoryInput: azdata.InputBoxComponent;
+
+	private numberInputType: azdata.InputBoxInputType = 'number';
```

Comment:
I'd just remove this, it isn't really necessary (just as easy to type it in below - the typings will ensure it's correct)